src/main/java/eci/ieti/leafOut/backEnd/controller/ParkController.java

```diff
@@ -30,10 +37,10 @@
 
 
     @GetMapping(path = "/{name}")
-    public ResponseEntity<?> getPlanByName(@PathVariable("name") String name){
+    public ResponseEntity<?> getPlanByName(@PathVariable("name") UUID parkName){
```

Comment:
`UUID` type is good for generating unique identifiers but it may face undesired effects when used in request or response payloads due to the marshalling/unmarshalling process